common/v2/features/ProtectTransaction/utils.ts

```diff
@@ -0,0 +1,70 @@
+import isNumber from 'lodash/isNumber';
+import { fromWei, totalTxFeeToWei, Wei } from 'v2/services/EthService/utils';
+import { IFormikFields } from 'v2/types';
+import {
+  PROTECTED_TX_FEE_PERCENTAGE,
+  PROTECTED_TX_FIXED_FEE_AMOUNT,
+  PROTECTED_TX_MIN_AMOUNT
+} from 'v2/config';
+import { ProtectTxError } from './types';
+
+export abstract class ProtectTransactionUtils {
```

Comment:
Please restrain from `class`.
Since it contains only static methods, the result would be same, but it may encourage a future to start adding instance methods and then it'll start being dangerous.
Prefer simple helper functions, and if you want encapsulation you can return a simple object or a factory.

Reply:
Its is actually marked as abstract, which means you can't create an instance of it.